### PR TITLE
10.14 support

### PR DESCRIPTION
**Pull request by cybertunnel**
Added support for 10.14's dark mode.

### CODE REVIEW

File: CHANGELOG.md.orig [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+* New exception in entitlements: `com.apple.security.temporary-exception.files.absolute-path.read-write` for `/private/tmp/SplashBuddy`
+
+## [1.1]
+
+### Added
+
+* Key `hideSidebar` to hide the sidebar and get a full html view
+* New behaviors for `Continue` button: Restart, Shutdown, Logout, Launch Application or Quit. (uses `continueAction` preferences key)
+* Hide Background with setting `hideBackground` to `true`. Eg. `SplashBuddy.app/Contents/MacOS/SplashBuddy -hideBackground true`
+<<<<<<< HEAD
```
Comment (Ethenyl) [inline CHANGELOG.md.orig]: This is some conflict code, this shouldn't be committed.

File: CHANGELOG.md.orig [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+### Fixed
+
+* Better automated testing with Travis
+* Compatible with Swift 4.0 and macOS 10.13
+* WKWebView now accepts keyboard input (can use forms)
+* Installer now better handle upgrades (thanks @elios and @scriptingosx)
+* SplashBuddy now handles multiple screens (thanks @jamfmatt)
+* Base.lproj renamed to en.lproj for 10.13 compatibility
+* Switched from legacy "launchctl load" to "enable, bootstrap, kickstart"
+<<<<<<< HEAD
```
Comment (Ethenyl) [inline CHANGELOG.md.orig]: Same here, this is conflict stuff.

File: Installer/payload/Library/Application Support/SplashBuddy/presentation.bundle/en.lproj/complete.html.orig [hunk 3]
```diff
@@ -0,0 +1,22 @@
+<html><head><meta charset="UTF-8"><body><div style="color:white;font-size:40px;border:0px;text-align:center;background-image:url('login_background.png');padding:20px;width:95%">
```
Comment (Ethenyl) [inline Installer/payload/Library/Application Support/SplashBuddy/presentation.bundle/en.lproj/complete.html.orig]: This is an `.orig` file, coming from a conflict, shouldn't be committed.

File: CHANGELOG.md.orig (hunk 1 above)
Comment (cybertunnel) [reply CHANGELOG.md.orig]: Removed this file, this snuck through.

File: CHANGELOG.md.orig (hunk 2 above)
Comment (cybertunnel) [reply CHANGELOG.md.orig]: Removed this file, this snuck through.

File: Installer/payload/Library/Application Support/SplashBuddy/presentation.bundle/en.lproj/complete.html.orig (hunk 3 above)
Comment (cybertunnel) [reply Installer/payload/Library/Application Support/SplashBuddy/presentation.bundle/en.lproj/complete.html.orig]: Removed this file, this snuck through.

**Comment by Ethenyl** [comment]
Based on the wrong branch, too many conflict due to huge changes in the codebase.

**Pull request closed without merging**